shared/params/config.go

```diff
@@ -96,6 +96,9 @@ type BeaconChainConfig struct {
 	EmptySignature            [96]byte      // EmptySignature is used to represent a zeroed out BLS Signature.
 	DefaultPageSize           int           // DefaultPageSize defines the default page size for RPC server request.
 	MaxPageSize               int           // MaxPageSize defines the max page size for RPC server respond.
+
+	// Slasher constants.
+	WeakSubjectivityPeriod uint64 // WeakSubjectivityPeriod defines the period were proof of stake network should validate block headers and attestations for slashable events.
```

Comment:
is this epoch? slot? we should probably mention it

Reply:
updated